### PR TITLE
Add last as chainable in lodash.d.ts

### PR DESCRIPTION
**Pull request by rosen-vladimirov**
`_.last()` is not defined correctly in lodash.d.ts, so it cannot be used in
chains. Add it (copy-paste the code from `_.first`).

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

File: definitions/lodash.d.ts [hunk 1]
```diff
@@ -1164,11 +1164,46 @@ declare module _ {
...
+        * @see _.last
+        * @param pluckValue "_.pluck" style callback value
+        **/
+        last(pluckValue: string): LoDashArrayWrapper<T>;
+
+        /**
+        * @see _.first
+        * @param whereValue "_.where" style callback value
+        **/
+        last<W>(whereValue: W): LoDashArrayWrapper<T>;
```
Comment (rosen-vladimirov) [inline definitions/lodash.d.ts]: I think this one is not working with lodash 3.x

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by Mitko-Kerezov** [comment]
:+1: and we could try replacing the whole deal from [DefinatelyTyped](https://github.com/borisyankov/DefinitelyTyped/blob/master/lodash/lodash.d.ts) in the `master` branch

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request merged**